tyk-data-plane/values.yaml

```diff
@@ -0,0 +1,377 @@
+    # annotation to that ingress controller.
+    ingress:
+      # if enabled, creates an ingress resource for the gateway
+      enabled: false
+
+      # specify ingress controller class name
+      className: ""
+
+      # annotations for ingress
+      annotations: {}
+
+      # ingress rules
+      hosts:
+        - host: chart-example.local
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+
+      # tls configuration for ingress
+      #  - secretName: chart-example-tls
+      #    hosts:
+      #      - chart-example.local
+      tls: []
+
+
+    # We usually recommend not to specify default resources and to leave this
+    # as a conscious choice for the user. This also increases chances charts
+    # run on environments with little resources, such as Minikube. If you do
+    # want to specify resources, uncomment the following lines, adjust them
+    # as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #  cpu: 100m
+    #  memory: 128Mi
+    resources: {}
+
+    # securityContext values for gateway pod
+    securityContext:
+      runAsUser: 1000
+      fsGroup: 2000
+
+    # node labels for gateway pod assignment
+    nodeSelector: {}
+
+    # tolerations for gateway pod assignment
+    tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+
+    # affinity for gateway pod assignment
+    affinity: {}
+
+    # extraEnvs is used to set gateway env variables
+    # - name: TYK_GW_HTTPSERVEROPTIONS_SSLINSECURESKIPVERIFY
+    #   value: "true"
+    extraEnvs:
+      - name: TYK_GW_SLAVEOPTIONS_RPCKEY
+        valueFrom:
+          secretKeyRef:
+            name: secrets-tyk-data-plane
+            key: rpcKey
+      - name: TYK_GW_SLAVEOPTIONS_APIKEY
+        valueFrom:
+          secretKeyRef:
+            name: secrets-tyk-data-plane
+            key: apiKey
+      - name: TYK_GW_SECRET
+        valueFrom:
+          secretKeyRef:
+            name: secrets-tyk-data-plane
+            key: APISecret
+      - name: TYK_GW_SLAVEOPTIONS_GROUPID
+        valueFrom:
+          secretKeyRef:
+            name: secrets-tyk-data-plane
+            key: groupID
+      - name: TYK_GW_SLAVEOPTIONS_CONNECTIONSTRING
+        valueFrom:
+          configMapKeyRef:
+            name: config-tyk-data-plane
+            key: rpcConnString
+      - name: TYK_GW_SLAVEOPTIONS_USESSL
+        valueFrom:
+          configMapKeyRef:
+            name: config-tyk-data-plane
+            key: rpcUseSSL
+      - name: TYK_GW_SLAVEOPTIONS_SSLINSECURESKIPVERIFY
+        valueFrom:
+          configMapKeyRef:
+            name: config-tyk-data-plane
+            key: rpcSslInsecureSkipVerify
+    # used to mount files to the gateway containers
+    #  - name: test
+    #    filename: test.txt
+    #    mountPath: /opt/test
+    mounts: []
+
+
+
+tyk-pump:
```

Comment:
can you please update tyk-pump chart according to the newest changes merged to the master? For example, `backend` field is an array of string at the moment.

Reply:
Fixed